test/spec_response.rb

```diff
@@ -419,20 +419,24 @@ def object_with_each.each
     header['content-length'].must_equal '9'
   end
 
-  it "correctly updates content-type when writing when initialized with body" do
-    obj = Object.new
-    def obj.each
-      yield 'foo'
-      yield 'bar'
-    end
-    ["foobar", ["foo", "bar"], obj].each do
```

Comment:
🗒️ This `.each` doesn't appear to be doing anything useful. There is no block param, so each iteration is identical. I rewrote this spec to simplify it.